### ISSUE TITLE
Asset Database crashes on parsing Package Dumper output

### ISSUE THREAD

**Issue by mirh** [ME3Tweaks/LegendaryExplorer#164]
**Describe the bug**
After logging which files the asset database was actually reading, I finally realized right before throwing the unhandled exception, it was reading from *\<file-it-was-looking-for>.txt* rather than .SFM

**Steps to reproduce the behavior:**
1. Dump the whole ME1 game in package dumper
2. Have output folder inside the game installation (I have it in Binaries for example)
3. Try to use Asset Database to open just about anything

**Expected behavior**
I'm not sure if there even is any value in parsing .txt files - regardless they shouldn't cause the program to get stuck. 

**Version information:**
4.1.1

**Stack trace**
```
FormatException: Not an ME1, ME2, ME3, or UDK package file.
   in ME3Explorer.Packages.MEPackageHandler.OpenMEPackage(String pathToFile, WPFBase wpfWindow, WinFormsBase winForm, Boolean forceLoadFromDisk)
   in ME3Explorer.AssetDatabase.AssetDB.ToggleRenderMesh()
   in ME3Explorer.AssetDatabase.AssetDB.lstbx_Meshes_SelectionChanged(Object sender, SelectionChangedEventArgs e)
   in System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   in System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   in System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   in System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   in System.Windows.Controls.ListBox.OnSelectionChanged(SelectionChangedEventArgs e)
   in System.Windows.Controls.Primitives.Selector.SelectionChanger.End()
   in System.Windows.Controls.Primitives.Selector.SetSelectedHelper(Object item, FrameworkElement UI, Boolean selected)
   in System.Windows.Controls.Primitives.Selector.NotifyIsSelectedChanged(FrameworkElement container, Boolean selected, RoutedEventArgs e)
   in System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   in System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   in System.Windows.Controls.ListBoxItem.OnSelected(RoutedEventArgs e)
   in System.Windows.Controls.ListBoxItem.OnIsSelectedChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
   in System.Windows.DependencyObject.OnPropertyChanged(DependencyPropertyChangedEventArgs e)
   in System.Windows.FrameworkElement.OnPropertyChanged(DependencyPropertyChangedEventArgs e)
   in System.Windows.DependencyObject.NotifyPropertyChange(DependencyPropertyChangedEventArgs args)
   in System.Windows.DependencyObject.UpdateEffectiveValue(EntryIndex entryIndex, DependencyProperty dp, PropertyMetadata metadata, EffectiveValueEntry oldEntry, EffectiveValueEntry& newEntry, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType)
   in System.Windows.DependencyObject.SetValueCommon(DependencyProperty dp, Object value, PropertyMetadata metadata, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType, Boolean isInternal)
   in System.Windows.DependencyObject.SetCurrentValueInternal(DependencyProperty dp, Object value)
   in System.Windows.Controls.ListBox.NotifyListItemClicked(ListBoxItem item, MouseButton mouseButton)
   in System.Windows.Controls.ListBoxItem.HandleMouseButtonDown(MouseButton mouseButton)
   in System.Windows.Controls.ListBoxItem.OnMouseLeftButtonDown(MouseButtonEventArgs e)
   in System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   in System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   in System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   in System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
   in System.Windows.UIElement.OnMouseDownThunk(Object sender, MouseButtonEventArgs e)
   in System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   in System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   in System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   in System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   in System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   in System.Windows.Input.InputManager.ProcessStagingArea()
   in System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   in System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   in System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   in System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   in System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   in MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   in MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   in System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   in System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by Mgamerz** [comment]
I would test if this is present in the 4.2 betas. There is a lot of changes
from 4.1 already present in 4.2.

**Comment by mirh** [comment]
Nope, I still get the error.

**Comment by Mgamerz** [comment]
@Kinkojiro Can you test this out in your code? It does not seem to filter out to *.<package>.

**Comment by Kinkojiro** [comment]
Fixed by 11d14c32
